src/ui/FieldValue/FieldValue.ts

```diff
@@ -410,6 +410,7 @@ export class FieldValue extends Component {
       const label = selected ? l('RemoveFilterOn', renderedFacetValue) : l('FilterOn', renderedFacetValue);
       new AccessibleButton()
         .withTitle(label)
+        .withLabel(label)
```

Comment:
Why is this addition needed?

Reply:
Complains in the console that there is no label.